apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryClientImplTest.cs

```diff
@@ -597,8 +597,7 @@ public void CreateJob_Labels()
         [Fact]
         public void ListTablesConversion()
         {
-            var listResource = new TableList.TablesData
-            {
+            var listResource = new TableList.TablesData {
```

Comment:
Please revert this change - it was fine as it was before.

Reply:
Ahh good eye!  I've got my VS formatting options set to put open {'s on the same line (probably not in alignment with the project).  I'll make sure to update them.  I'm new to contributing on github in general so I appreciate the help!  Thanks @jskeet!

Reply:
Let me know if you need a hand with the git aspects of it. Once you've created an extra commit for this bit, if you use `git rebase -i master` then change the second and third lines to have "f" at the start instead of "pick", it will squash them all together. Then just run `git push -f` to update the PR.